chemprop/models/model.py

```diff
@@ -10,7 +10,7 @@
 
 from chemprop.data import TrainingBatch, BatchMolGraph
 from chemprop.nn.metrics import Metric
-from chemprop.nn import MessagePassing, Aggregation, Predictor, LossFunction
+from chemprop.nn import MessagePassing, Aggregation, Predictor, LossFunction, MveFFN
```

Comment:
```suggestion
from chemprop.nn import MessagePassing, Aggregation, Predictor, LossFunction
```
This isn't used now that you moved those changes to another PR. Last fix and then I will merge this PR in.